### ISSUE TITLE
2.0rc中dygraph下的gn在rocm平台下loss出错

### ISSUE THREAD

**Issue by kali20gakki** [PaddlePaddle/PaddleDetection#2928]
环境：paddlepaddle-rocm  2.0.2-rocm-post 
           ppdet                        2.0rc0

问题：在rocm平台下，定位到CascadeHead中使用CascadeXConvNormHead时，如果norm_type设为gn，则loss第一步就为nan，学习率设置正确，输出log如下：
ppdet.engine.callbacks INFO: Epoch: [0] [    0/13653] learning_rate: 0.000001 loss_rpn_cls: 0.692172 loss_rpn_reg: 0.012241 loss_bbox_cls_stage0: 97998976.000000 loss_bbox_reg_stage0: 0.039062 loss_bbox_cls_stage1: 0.000000 loss_bbox_reg_stage1: nan loss_bbox_cls_stage2: 0.000000 loss_bbox_reg_stage2: nan loss: nan eta: 49 days, 12:08:43 batch_cost: 8.7024 data_cost: 0.0016 ips: 0.1149 images/s

相同的配置在nvidia环境就没问题， 在rocm平台下，将norm_type设为bn，则loss正常

**Comment by yghstill** [comment]
@kali20gakki 目前gn模型在rocm中支持的不是很好，我们将排期支持下，请先使用bn吧

**Comment by qili93** [comment]
@kali20gakki 我们在最新的develop分支代码上验证了一下已经没有loss nan了，你如果是自己的环境，可以尝试使用一下“源码编译”从develop分支或者release/v2.1分支编一个wheel包出来，或者再等一周左右很快会有v2.1.0的ROCM新包发布。

如果你用的是AIStudio的环境，那也要稍等几天，AIStudio那边很快会上ROCM v2.1.0的新包，这个新包里面是没有loss nan的问题的。
